modules/nf-core/bwamem2/index/meta.yml

```diff
@@ -39,13 +39,27 @@ output:
           pattern: "*.{0123,amb,ann,bwt.2bit.64,pac}"
           ontologies:
             - edam: "http://edamontology.org/data_3210" # Genome index
-  versions:
-    - versions.yml:
-        type: file
-        description: File containing software versions
-        pattern: "versions.yml"
-        ontologies:
-          - edam: http://edamontology.org/format_3750 # YAML
+  versions1:
+    - - ${task.process}:
+          type: string
+          description: The process the versions were collected from
+      - bwa-mem2:
+          type: string
+          description: The tool name
+      - bwa-mem2 version 2>&1 | tail -1:
+          type: string
+          description: The command used to generate the version of the tool
+topics:
+  - versions:
+      - - process:
+            type: string
+            description: The process the versions were collected from
+        - tool:
+            type: string
+            description: The tool name
+        - version:
+            type: string
+            description: The version of the tool
```

Comment:
this looks good! 
but if we want to automate it somehow, the values of the topic channel should be the same that we add for `versions1`: `${task.process}`, `bwa-mem2` and `bwa-mem2 version 2>&1 | tail -1`.
It doesn't look as nice, but it's the only way we can automate the generation of the meta.yml